### ISSUE TITLE
`TabContainer` lacks `tab_hovered` signal

### ISSUE THREAD

**Issue by git2013vb** [godotengine/godot#51618]
### Godot version

3.4 beta3 mono

### System information

Debian 10

### Issue description

I tried to use TabContainer but I have errors when I try to trigger some signals:
```
E 0:00:02.180   emit_signal: Error calling method from signal 'tab_selected': 'Control(Control.cs)::_on_TabContainer_tab_selected': Method not found..
  <C++ Source>  core/object.cpp:1238 @ emit_signal()

```
```
E 0:00:04.105   emit_signal: Error calling method from signal 'tab_changed': 'Control(Control.cs)::_on_TabContainer_tab_changed': Method not found..
  <C++ Source>  core/object.cpp:1238 @ emit_signal()

```
While the `_on_TabContainer_pre_popup_pressed` signal is missing.
Also we don't have a signal when mouse hover a single tab + No indications about its use in documentation

### Steps to reproduce

Attached min project

### Minimal reproduction project

[TestTabContainer.zip](https://github.com/godotengine/godot/files/6981773/TestTabContainer.zip)

**Comment by kleonc** [comment]
Your `Control(Control.cs)::_on_TabContainer_tab_selected` and `Control(Control.cs)::_on_TabContainer_tab_changed` methods doesn't match signatures of the signals you've connected them to, hence the errors (yeah, error message isn't straightforward, see: #35910). Both `tab_changed` and `tab_selected` signals [have `int tab` parameter](https://docs.godotengine.org/en/3.4/classes/class_tabcontainer.html#signals). Adding such parameter to the methods you've connected to these signals will make them be connected properly (they'll be found).

**Comment by git2013vb** [comment]
> Your `Control(Control.cs)::_on_TabContainer_tab_selected` and `Control(Control.cs)::_on_TabContainer_tab_changed` methods doesn't match signatures of the signals you've connected them to, hence the errors (yeah, error message isn't straightforward, see: #35910). Both `tab_changed` and `tab_selected` signals [have `int tab` parameter](https://docs.godotengine.org/en/3.4/classes/class_tabcontainer.html#signals). Adding such parameter to the methods you've connected to these signals will make them be connected properly (they'll be found).

Thank you for your hint.  1 question before close this issue:  how I can fire a signal when mouse hover a single tab name? (Not the entire Control).   My idea is to show a text when hover a specific tab

**Comment by kleonc** [comment]
>how I can fire a signal when mouse hover a single tab name?

Seems like currently `TabContainer` doesn't notify about such thing in any way. You could use `Tabs` control instead which has [`tab_hover` signal](https://docs.godotengine.org/en/3.4/classes/class_tabs.html#signals). But note that in such case you'd need to manually control contents of the tabs.

**Comment by git2013vb** [comment]
If this is intended we can close this issue while if the current plan is to add signals to tabcontainer wee can keep it open.. I let you to decide. 
Thanks

**Comment by kitbdev** [comment]
Implemented in #76374 as well as `tab_clicked` and `active_tab_rearranged` signals.
